android/java/org/chromium/chrome/browser/shields/AddCustomFilterListsFragment.java

```diff
@@ -48,10 +50,26 @@ public void onActivityCreated(Bundle savedInstanceState) {
 
     private void setData() {
         initFilterListAndroidHandler();
+        Button addBtn = getView().findViewById(R.id.btn_add);
         TextInputEditText urlEditText = getView().findViewById(R.id.enter_url_edittext);
         urlEditText.requestFocus();
+        urlEditText.addTextChangedListener(new TextWatcher() {
+            @Override
+            public void afterTextChanged(Editable s) {}
+
+            @Override
+            public void beforeTextChanged(CharSequence s, int start, int count, int after) {}
+
+            @Override
+            public void onTextChanged(CharSequence s, int start, int before, int count) {
+                if (s.toString().trim().length() > 0) {
```

Comment:
```suggestion
                if (s.toString().trim().length() > 0) {
```
addBtn.setEnabled(s.toString().trim().length()>0 ? true : false);